test_files/samples/rename_headers_label.txt

```diff
@@ -0,0 +1,5 @@
+lid,LID
+bundle,bundle_name
+bundle_lid,bundle_LID
+filename,file_name
+filepath,path_to_file
```

Comment:
The reason my problem is not being caused by the test is that you're only testing the added extra fields, not the normal fields taken from labels where type lookup is required.